docs/collab/133.md

```diff
@@ -1,4 +1,28 @@
 # 5	Weitere Open-Science-Themen
 ## 5.1 Berücksichtigung von Open Source in Förderinstrumenten
 
-[Format: Stand heute – Problem – Vorschlag einer Massnahme]
+Im Rahmen der in den letzten zwei Jahren etablierten [Community of Practice "Open Source"](https://bfh.github.io/contact#bfhchopensource) haben wir aus Sicht (AHB, HAFL, TI, W und Vizerektorat) erste strategische [Anlaufpunkte zum Thema Open Source](http://bfh.ch/opensource) an der BFH aufbauen können.
+
+Analog zu 4.1 stellen auch wir fest, dass die "Mitarbeit an" und "Freigabe von" quelloffenen Systemen bisher nicht als eigenständige (und gleichwertige) Leistung in der Forschungsinitiative berücksichtigt wird (s. [#52](https://github.com/bfh/opensource/issues/52)).
+
+Nebst einem **klaren Committment zur präferierten ([open-by-default](https://en.wikipedia.org/wiki/Open_by_default)) [Nutzung (1), Mitarbeit (2) & Freigabe (3)](https://bfh.github.io) von Artefakten** unter freien und offenen [Lizenzen nach SPDX](https://spdx.org/licenses/): 
```

Comment:
Vorschlag; Zur leichteren Lesbarkeit in zwei abgeschlossene Sätze zerlegen, open-by-default Klammer evtl. hinter schieben:
Wir schlagen ein **klares Committment zur präferierten [Nutzung (1), Mitarbeit (2) & Freigabe (3)](https://bfh.github.io) von Artefakten** ([open-by-default](https://en.wikipedia.org/wiki/Open_by_default)) unter freien und offenen [Lizenzen nach SPDX](https://spdx.org/licenses/) vor:

Reply:
See 9c484e8